CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # ChangeLog
+## 1.12.0
+### Removed
+- libmongocrypt is no longer published in the PPA for RHEL 6. libmongocrypt may instead be built from source on RHEL 6, but support for RHEL 6 will be dropped in a future release.
```

Comment:
Not sure if the term "PPA" is correct here, since that usually refers to Canonical's Debian package hosting (unless we have our own "PPA" thing?)

Reply:
I am glad you asked. I expect "PPA" is misleading. Updated to "MongoDB package repository".